ext_emconf.php

```diff
@@ -7,7 +7,7 @@
     'author_email' => 'projects@jweiland.net',
     'author_company' => 'jweiland.net',
     'state' => 'stable',
-    'version' => '8.3.1',
+    'version' => '8.3.2',
```

Comment:
Don't forget to update version in Documentation settings, too

Reply:
Done